utils/clusterutils.go

```diff
@@ -99,9 +107,19 @@ func getDaemonset(deployment *appsv1.Deployment) *appsv1.DaemonSet {
 				Spec: corev1.PodSpec{
 					NodeSelector:                  cfg.NodeSelector,
 					TerminationGracePeriodSeconds: &terminationGracePeriodSeconds,
-					Containers:                    getContainers(),
-					ImagePullSecrets:              imgPullSecrets,
-					Affinity:                      cfg.Affinity,
+					InitContainers: []corev1.Container{{
+						Name:            "copy-sleep",
+						Image:           cfg.ImagePullerImage,
+						ImagePullPolicy: corev1.PullAlways,
+						Command:         []string{"/bin/sh"},
+						Args:            []string{"-c", "cp bin/sleep kip/sleep"},
```

Comment:
A safer bet here would be to use absolute paths:
```suggestion
						Args:            []string{"-c", "cp /bin/sleep /kip/sleep"},
```
in case the workdir for the container changes.

Reply:
Thank you, updated in my latest commit